enterprise/server/cmd/ci_runner/bazel_wrapper/bazel_wrapper.go

```diff
@@ -0,0 +1,185 @@
+package main
```

Comment:
I know this is WIP but just as a quick check, what sorts of custom bazel args does this wrapper need to handle? I was hoping we could get away with having the ci_runner generate an sh script called `bazel` which just contains

```
#!/usr/bin/env bash
exec bazel {{startup_options}} "$@"
```

and then for any non-startup options we could write those to the generated .bazelrc?

Reply:
The most complex startup option it needs to handle is the one that points to the generated .bazelrc. We need to check if there's a workspace .bazelrc and add that explicitly as a --bazelrc so that it takes precedence

Sounds good about adding non-startup options to the generated .bazelrc